osquery/tables/applications/posix/docker.cpp

```diff
@@ -557,17 +557,30 @@ QueryData genContainerPorts(QueryContext& context) {
  */
 QueryData genContainerProcesses(QueryContext& context) {
   QueryData results;
+  std::string ps_args;
+
   for (const auto& id : context.constraints["id"].getAll(EQUALS)) {
     if (!checkConstraintValue(id)) {
       continue;
     }
 
     pt::ptree container;
-    Status s =
-        dockerApi("/containers/" + id + "/top?ps_args=axwwo%20" +
-                      "pid,state,uid,gid,euid,egid,suid,sgid,rss,vsz,etimes,"
-                      "ppid,pgrp,nlwp,nice,user,time,pcpu,pmem,comm,cmd",
-                  container);
+
+    if (isPlatform(PlatformType::TYPE_OSX)) {
```

Comment:
Would this look better as a ternary?

Reply:
Likely not, but only because there's three conditions I'm seeing.